examples/ethereum/foundry_test.go

```diff
@@ -36,7 +37,7 @@ func TestEthereum(t *testing.T) {
 	ctx := context.Background()
 
 	// Get default ethereum chain config for anvil
-	anvilConfig := ethereum.DefaultEthereumAnvilChainConfig("ethereum")
+	anvilConfig := foundry.DefaultEthereumAnvilChainConfig("ethereum")
```

Comment:
note to self: 
- add release note that we support both foundry & geth now
- modifications to local-ic required?